lib/buildFile.js

```diff
@@ -11,13 +11,13 @@
  * and limitations under the License.
  */
 
-var path = require('path'),
-  fs = require('fs-extra'),
-  chalk = require('chalk'),
-  filterProperties = require('./filterProperties'),
-  GroupMessages = require('./utils/groupMessages');
+import path from '@bundled-es-modules/path-browserify';
+import chalk from 'chalk';
+import { fs } from '../index.js';
```

Comment:
Does this create a circular dependency because index.js imports this file too?

Reply:
Yes, there's a few actually in the library that I spotted in the configurator project when bundling stuff for prod:

```
node_modules/style-dictionary/index.js -> node_modules/style-dictionary/lib/extend.js -> node_modules/style-dictionary/index.js
node_modules/style-dictionary/index.js -> node_modules/style-dictionary/lib/extend.js -> node_modules/style-dictionary/lib/utils/combineJSON.js -> node_modules/style-dictionary/index.js
node_modules/style-dictionary/index.js -> node_modules/style-dictionary/lib/common/transforms.js -> node_modules/style-dictionary/lib/utils/convertToBase64.js -> node_modules/style-dictionary/index.js
...and 6 more
```

Maybe this PR is a good time to deal with them all :) even though most bundlers can deal with circular dependencies, the order of imported modules might matter, so circular deps are still an anti-pattern that should preferably be dealt with

Reply:
Fixed the circular deps in the most recent commit :) tested with Rollup bundle -> no warnings whatsoever now.